Sources/Core/AWSSDKHTTPAuth/Sources/AWSSDKHTTPAuth/AWSSigV4Signer.swift

```diff
@@ -273,3 +274,18 @@ extension SigningConfig {
         }
     }
 }
+
```

Comment:
Below are extensions for taking a`S3ExpressIdentity` & making it into a regular `AWSCredentialIdentity` for the purposes of code signing.

Reply:
Method is private because it is not used anywhere else (nor should it.)

`S3ExpressIdentity` and `AWSCredentialIdentity` have basically all the same fields, but are separate & distinct types because they are used only in specific, mutually exclusive scenarios.